crates/relayer-types/src/applications/transfer/msgs/transfer.rs

```diff
@@ -41,6 +41,8 @@ pub struct MsgTransfer<C = Coin> {
     /// Timeout timestamp relative to the current block timestamp.
     /// The timeout is disabled when set to 0.
     pub timeout_timestamp: Timestamp,
+    /// optional memo
+    pub memo: String,
```

Comment:
Since this is optional, let's rather use an Option/None here instead of an empty string.

Reply:
Since the memo field is a String and not an option in the proto derived struct `ibc_proto::ibc::applications::transfer::v1::MsgTransfer`, https://github.com/cosmos/ibc-proto-rs/blob/v0.28.0/src/prost/ibc.applications.transfer.v1.rs#L32, should the memo field be parsed and set to `None` if it is empty when converting it into the `MsgTransfer` defined in the `relayer-types` crate ?

Reply:
Yes that would make the most sense imho.